src/tests/regression/rhbz1689429.at

```diff
@@ -0,0 +1,9 @@
+FWD_START_TEST([rich rule invalid priority])
+AT_KEYWORDS(rich rhbz1689429)
+
+FWD_CHECK([--add-rich-rule='rule priority=foo accept'], 122, [],
```

Comment:
Can you add the `--permanent` flag here? That'll magically get us `firewall-offline-cmd` coverage as well. Then you can change the `FWD_RELOAD` to check for the runtime error.
i.e.
```
FWD_RELOAD([122], [ignore], [ignore], [251])
```

Reply:
As you noted on IRC.. the regression tests aren't ran for `firewall-offline-cmd`. Sorry for misleading. We should still add `--permanent` flag though.